src/Sentry.Unity/Integrations/ITime.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using UnityEngine;
+
+namespace Sentry.Unity
+{
+    internal interface ITime
+    {
+        DateTime Now { get; }
```

Comment:
```suggestion
        DateTimeOffset Now();
```

Reply:
Can be replaced by using `Sentry.Infrastructure.ISystemClock.GetUtcNow()`

And; https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/Infrastructure/SystemClock.cs